btell_main/forms/login.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.auth import forms as auth_forms
+from django.contrib.auth import views as auth_views
+from django.contrib.auth import authenticate, login
+
+from django import forms
+from django.urls import reverse_lazy
```

Comment:
Per new style guide, we should import `from django import url`, then use `url.reverse_lazy()`.

Can be prettified later.